src/models/models.controller.ts

```diff
@@ -11,7 +11,7 @@ export let bpmn2sol = xml =>
     let abi = "";
     const cont = parseModel(modelInfo);
     return cont.then(() => {
-      const input = {};
+      let input = {};
```

Comment:
Does this really need to be a `let`? We can change the object's properties in a `const`, we just can't change the pointer, right?

Reply:
Yes I was trying few other things with `input` so changed to `let` but forgot to revert back. Will change it 👍